db/migrate/20150207064508_drop_media_item.rb

```diff
@@ -0,0 +1,5 @@
+class DropMediaItem < ActiveRecord::Migration
+  def change
+    drop_table :media_items
```

Comment:
Maybe you can get that to work:  I just thought it was easier to drop the
table and create a new one, since the fields weren't the same anywya.

On Tue, Feb 10, 2015 at 10:15 PM, Aaron Panchal <notifications@github.com>
wrote:

> In db/migrate/20150207064508_drop_media_item.rb
> <https://github.com/hammersmith/bookends/pull/7#discussion_r24476082>:
>
> > @@ -0,0 +1,5 @@
> > +class DropMediaItem < ActiveRecord::Migration
> > +  def change
> > +    drop_table :media_items
>
> Why didn't we rename?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammersmith/bookends/pull/7/files#r24476082>.
>



-- 

​--------
​-------===============########===============---------------​


Want to quickly check
​a ​
system status
​ or​
report
​a problem to the IT team?
Use http://justme.westmont.edu
​
​
Have a problem that requires tracking and IT email response?
Use http://mayday.westmont.edu​